src/events.js

```diff
@@ -55,11 +56,17 @@ module.exports = function(ctx) {
     currentMode.mousedown(event);
   };
 
+  events.drawChanged = function(event) {
+    featuresChanged = event.features;
+  }
+
   events.mouseup = function(event) {
     mouseDownInfo.isDown = false;
     var target = getFeatureAtAndSetCursors(event, ctx);
     event.featureTarget = target;
 
+    ctx.map.fire('draw.modifiedFeature', {features: featuresChanged});
```

Comment:
in https://github.com/mapbox/mapbox-gl-draw/pull/358/files#diff-4607e46b15da8ae44c4b47730c6d5963R48 we need to reset featuresChanged to [] or else we keep emitting the same features as changed.